ns/init.go

```diff
@@ -0,0 +1,15 @@
+package ns
+
+var (
+	basePath string
```

Comment:
Again my preference is to move the changes in this file to `osl/namespace_linux.go` and construct the `netns` base path using `exec root` setting to keep it linux specific and create an exported function in `osl` package to set the basepath.